### PR TITLE
Add context propagation manual instrumentation to dt-otel codelab

### PR DESCRIPTION
**Pull request by spencerhank**

### CODE REVIEW

**Comment by spencerhank** [comment]
Still very much a work in progress. Pushing code before extended out of office time.

File: markdown/dt-otel/dt-otel.md [hunk 1]
```diff
@@ -779,6 +808,176 @@ The second span was generated by the PubSub+ Broker when the message was receive
...
+        sendSpan.setStatus(StatusCode.ERROR, e.getMessage()); //Set span status as ERROR/FAILED
+    } finally {
+        sendSpan.end(); //End sendSpan. Span data is exported when span.end() is called.
+    }
+}
+```
+
+We've provided a jar containing the manual implementation code from above, to run the jar and publish messages to the broker, execute the command below from the tracing-codelab directory.
+```bash
+java -jar solace-samples-jcsmp-publisher-manual-instrumentation-1.0-SNAPSHOT.jar tcp://localhost:55557 default default default
```
Comment (spencerhank) [inline markdown/dt-otel/dt-otel.md:885]: File is currently available [here](https://solacesystems-my.sharepoint.com/personal/hank_spencer_solace_com/_layouts/15/onedrive.aspx?FolderCTID=0x0120000F184F3829FEEA4782B84E73C4226B07&id=%2Fpersonal%2Fhank%5Fspencer%5Fsolace%5Fcom%2FDocuments%2FDT%20OTEL%20Code%20Lab%2Fmanual%2Dinstrumentation%2Dpublisher%2Ftarget&view=0). We'll need to copy the final version of both the unpackaged code and the jar to the tracing-codelab zip that is currently downloaded at the start of the codelab.

Unpackaged code can be ran as a java class. I'm assuming we'll want to provide the users with a .jar file so they don't have to package anything up.

File: markdown/dt-otel/dt-otel.md [hunk 2]
```diff
@@ -779,6 +808,176 @@ The second span was generated by the PubSub+ Broker when the message was receive
...
+        }
+    } finally {
+     receiveSpan.end(); //End receiveSpan. Span data is exported when span.end() is called.
+    }
+}
+```
+
+We've provided a jar containing the manual implementation code from above, to run the jar and subscribe to the `q` queue on the broker, execute the command below from the tracing-codelab directory.
+```bash
+java -jar solace-samples-jcsmp-subscriber-manual-instrumentation-1.0-SNAPSHOT.jar tcp://localhost:55557 default default default
```
Comment (spencerhank) [inline markdown/dt-otel/dt-otel.md:970]: File is currently available in the target directory [here](https://solacesystems-my.sharepoint.com/personal/hank_spencer_solace_com/_layouts/15/onedrive.aspx?FolderCTID=0x0120000F184F3829FEEA4782B84E73C4226B07&id=%2Fpersonal%2Fhank%5Fspencer%5Fsolace%5Fcom%2FDocuments%2FDT%20OTEL%20Code%20Lab%2Fmanual%2Dinstrumentation%2Dsubscriber&view=0). We'll need to copy the final version of both the unpackaged code and the jar to the tracing-codelab zip that is currently downloaded at the start of the codelab.

Unpackaged code can be ran as a java class. I'm assuming we'll want to provide the users with a .jar file so they don't have to package anything up.

File: codelabs/dt-otel/img/6f508b06c0f5da59.png
Comment (spencerhank) [inline codelabs/dt-otel/img/6f508b06c0f5da59.png:1]: @TamimiGitHub - reminder to update the publisher app and screenshots to create a SEND span, not PROCESS

**Comment by TamimiGitHub** [comment]
Made modifications to clean up the codelab. Added/fixed the following: 
- Made sure there was a `Duration` for every section
- Typos
- Added all the code samples in a [github repo](https://github.com/TamimiGitHub/solace-dt-demo), no more reference to downloading zip folder
- Fixed Manual and Auto sections
- Added a simple arch diagram

Ready to be merged from my end.

File: markdown/dt-otel/dt-otel.md (hunk 1 above)
Comment (TamimiGitHub) [reply markdown/dt-otel/dt-otel.md:948]: We will also need readme instructions on how to compile the java code (maven or gradle)

File: codelabs/dt-otel/img/6f508b06c0f5da59.png
Comment (TamimiGitHub) [reply codelabs/dt-otel/img/6f508b06c0f5da59.png:1]: I changed it in the code, need to change it in the screenshot. Having issues compiling and running the manual inst application

Comment (TamimiGitHub) [reply codelabs/dt-otel/img/6f508b06c0f5da59.png:1]: done

**Pull request merged**